wds-blocks.php

```diff
@@ -90,3 +90,26 @@ function register_block() {
 	}
 }
 add_action( 'init', __NAMESPACE__ . '\register_block' );
+
+/**
+ * Adds a WDS Block category to the Gutenberg category list.
+ *
+ * @param array  $categories The existing categories.
+ * @param object $post The current post.
+ * @return array The updated array of categories.
+ * @author WebDevStudios
+ * @since 2.0.0
+ */
+function register_block_category( $categories, $post ) {
+
+	return array_merge(
+		$categories,
+		array(
+			array(
+				'slug'  => 'wds-blocks',
+				'title' => esc_html__( 'WDS Blocks', 'wdsblocks' ),
```

Comment:
BEE question: should it be `esc_html__()` or `__()` here?

Reply:
Good question. This was a direct copy and paste from the `wd_s` theme.

Reply:
It should be `esc_html__()` 

With PHP, we **should not** be using `__()`, while the possibility is remote, it possible for translations to contain malicious code and using `esc_html__()` ensures they'll be escaped.